runtime/src/main/resources/messages.properties

```diff
@@ -14,5 +14,9 @@
 # limitations under the License.
 #
 
-io.syndesis.credential.RelativeUri.message = Given URI value must be relative
-io.syndesis.model.validation.UniqueProperty.message = Value `${validatedValue[value]}` for property `{value}` is not unique
+io.syndesis.credential.RelativeUri.message          = Given URI value ('{value}') must be relative
+io.syndesis.model.validation.UniqueProperty.message = Value '${nonUnique}' is not unique
```

Comment:
Not an BeanValidation connoisseur at all, but why are parameters sometimes referenced with `${...}` and sometimes with `{...}` ? Reminds me so to the JSF fiasco (`#{...}` anyone ?)